src/index.js

```diff
@@ -260,5 +260,3 @@ LazyLoad.defaultProps = {
 };
 
 export default LazyLoad;
-
-export lazyload from './decorator';
```

Comment:
As far as I can tell, the default key from this file (`export default LazyLoad`) is being overwritten by the default key being exported from `'./decorator'`.  By importing from `react-lazyload/decorator` directly, we don't have contention for the key.